code/modules/cm_aliens/structures/trap.dm

```diff
@@ -217,6 +217,20 @@
 			to_chat(X, SPAN_XENONOTICE("[src] is occupied by a child."))
 			return XENO_NO_DELAY_ACTION
 
+	if(X.a_intent == INTENT_HARM)
+		to_chat(X, SPAN_XENONOTICE("You start tearing away at the hole."))
+		xeno_attack_delay(X)
+		if(!do_after(X, 30, INTERRUPT_NO_NEEDHAND, BUSY_ICON_HOSTILE, src))
```

Comment:
Use the time macros. E.g. `3 SECONDS` See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#use-our-time-defines